src/main.js

```diff
@@ -63,6 +65,7 @@ function toggleStarOffData() {
       savedIdeas[i].star = false;
     }
   }
+  saveIdeaToStorage()
```

Comment:
Updates the savedIdeas array and saves this new array to localStorage, overwriting what is currently saved there.